#!/usr/bin/python
PACKAGE = "wheel_leg_fsm"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("OUTPUT_ratio", double_t, 0, "output ratio", 0.0, 0, 1)
gen.add("dx_balance", double_t, 0, "dx_balance", 0.007, -0.02, 0.02)
gen.add("ratio1", double_t, 0, "ratio1", 15.0, 0, 20)
gen.add("ratio2", double_t, 0, "ratio2", 0.0, 0, 1.0)
gen.add("ratio3", double_t, 0, "ratio3", 0.0, 0, 0.4)

exit(gen.generate(PACKAGE, "wheel_leg_fsm", "Manual"))
